theatre/core/src/propTypes/index.ts

```diff
@@ -138,6 +138,45 @@ export const compound = <Props extends UnknownShorthandCompoundProps>(
   return config
 }
 
+/**
+ * An image prop type
+ *
+ * @example
+ * Usage:
+ * ```ts
+ *
+ * // with a label:
+ * const obj = sheet.object('key', {
+ *   url: t.image({
+ *     label: 'texture'
+ *   })
+ * })
+ * ```
+ *
+ * @param opts - Options (See usage examples)
+ */
+export const image = (
+  opts: {
+    label?: string
+    interpolate?: Interpolator<string>
+  } = {},
+): PropTypeConfig_Image => {
+  if (process.env.NODE_ENV !== 'production') {
+    validateCommonOpts('t.image(defaultValue, opts)', opts)
+  }
+
+  return {
+    type: 'image',
+    // FIXME
+    default: undefined as $FixMe,
```

Comment:
`default` value cannot be unassignable to `valueType`.

Reply:
so `default` should be `{type: 'Image', path: ''}` or something like that

Reply:
Yes, we can require default values anyway now that asset ids are user-managed. Although using {type: 'Image', path: ''} instead of a simple string type would make it the first prop type to somehow encode the theatre type in the value instead of the theatre type being determined at runtime through sheet.object() calls so I'm skeptical about that part.

Reply:
Yeah it is a first. We haven't encoded values like this because, so far all value types have been infer-able by their JSON type. The image type is the first one for which we need to encode additional information so it won't be mistaken for another type, like a string, or a hypothetical "AudioFile" type. So I say go for `{type: 'Image', path: string}`.